BytexDigital.Blazor.Components.CookieConsent/CookieConsentOptions.cs

```diff
@@ -162,26 +173,31 @@ public class CookieConsentOptions
                 TitleText = new()
                 {
                     ["en"] = "Strictly necessary cookies",
-                    ["de"] = "Umbedingt notwendige Cookies"
+                    ["de"] = "Umbedingt notwendige Cookies",
+                    ["nl"] = "Strict noodzakelijke cookies"
                 },
                 DescriptionText = new()
                 {
                     ["en"] = "These cookies are essential for the proper functioning of this website. They do not contain personal data and are not used to track you.",
-                    ["de"] = "Diese Cookies sind umbedingt notwendig für die Nutzung dieser Webseite. Sie enthalten keine personenbezogenen Daten und werden nicht für Tracking verwendet."
+                    ["de"] = "Diese Cookies sind umbedingt notwendig für die Nutzung dieser Webseite. Sie enthalten keine personenbezogenen Daten und werden nicht für Tracking verwendet.",
+                    ["nl"] = "deze cookies zijn essentieel voor het goed functioneren van deze website. Ze bevatten geen persoonlijke gegevens en worden niet gebruikt om u te volgen."
```

Comment:
Is this supposed to not be capitalized?

Reply:
You're wright, it is supposed to be capitalized